sycl/include/CL/sycl/handler.hpp

```diff
@@ -193,16 +193,14 @@ class handler {
   /// It's expected that the method is the latest method executed before
   /// object destruction.
   ///
-  /// @return a SYCL event object representing the command group.
-  event finalize();
-
-  /// Saves streams associated with this handler.
-  ///
-  /// Streams are then forwarded to command group and flushed in the scheduler.
-  ///
-  /// @param Stream is a pointer to SYCL stream.
-  void addStream(shared_ptr_class<detail::stream_impl> Stream) {
-    MStreamStorage.push_back(std::move(Stream));
+  /// @param Payload contains the code location of user code
+  /// @return a SYCL event object representing the command group
+  event finalize(const cl::sycl::detail::code_location &Payload = {});
+
+  // Save streams associated with this handler. Streams are then forwarded to
+  // command group and flushed in the scheduler.
```

Comment:
I believe the original layout was done on purpose. @alexbatashev

Reply:
True, the original comment was a piece of doxygen documentation. @tovinkere please get it back.

Reply:
Done!